src/appshell/qml/DevTools/DevToolsPage.qml

```diff
@@ -94,7 +94,7 @@ DockPage {
         }
     ]
 
-    central: settingsComp
+    central: extensionsComp
```

Comment:
Did you forget to reset this change?

Reply:
For now I constantly need this page, when I finish the work completely, I can return the settings..